ipahttp/ipahttp.py

```diff
@@ -73,9 +73,14 @@ def config_show(self):
 
         return results
 
-    def group_add(self, group, gidnumber, description=None):
-        m = {'method': 'group_add', 'item': [group], 'params': {'all': True,
-             'gidnumber': gidnumber, 'description': description}}
+    def group_add(self, group, description=None):
```

Comment:
I don't think that dropping the possibility to choose a gid to be a good idea. There are many scenarios when it's important to be able to set it to an expected value. That said, I do see the convenience of not having to set it, how about making it an optional parameter instead?